UI/signup.html

```diff
@@ -7,6 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" media="screen" href="static/main.css" />
     <script src="static/main.js"></script>
+    <!-- <script src="static/js/signup.js"></script> -->
```

Comment:
Try not to include commented out code in the PR.

Reply:
Alright, I am going to clear this in my next commit.